functions/pom.xml

```diff
@@ -0,0 +1,109 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This parent pom.xml doesn't seem necessary

Reply:
Deleted it.